### DISCUSSION TITLE
How to deploy manifest files after deleting resources

### DISCUSSION THREAD

**Discussion by KentonParton** [aws/copilot-cli#5942] [Q&A]
I have gone through the process of initializing copilot with an app/ env and deploying the infra to AWS. 

I then deleted the application/ env resources from AWS with `copilot app delete`. I still have the manifest files locally. 

```
❯ tree .
.
└── copilot
    ├── environments
    │   └── dev
    │       └── manifest.yml
    └── hocuspocus
        └── manifest.yml
```

How do I redeploy all copilot infrastructure?

When I try to deploy them I get an error:
```
❯  copilot deploy --app hocuspocus --env dev
The "copilot" directory is not associated with an application.
Run `copilot app init` to create or use an application.
✘ couldn't find an application associated with this workspace
```

**Comment by iamhopaul123** [comment]
Hello @KentonParton. We don't delete your local files when you run `app delete`. However, all the online resources that we created would be gone and that's why you saw that error msg. In order to redeploy, you would need to go through the initialization process again.
